File: deno/src/wxwork/wxw_app_token/wxw_app_token.resolver.ts
import type {
  WxwGetConfigSignature,
} from "/gen/types.ts";

/** 通过 appid, agentid, url 生成企业签名 */
export async function wxwGetConfigSignature(
  appid: string,
  agentid: string,
  url: string,
): Promise<WxwGetConfigSignature> {
  
  const {
    wxwGetConfigSignature,
  } = await import("./wxw_app_token.service.ts");
  
  const res = await wxwGetConfigSignature(appid, agentid, url);
  
  return res;
}

/** 通过 appid, agentid, url 生成应用签名 */
export async function wxwGetAgentConfigSignature(
  appid: string,
  agentid: string,
  url: string,
): Promise<WxwGetConfigSignature> {
  
  const {
    wxwGetAgentConfigSignature,
  } = await import("./wxw_app_token.service.ts");
  
  const res = await wxwGetAgentConfigSignature(appid, agentid, url);
  
  return res;
}

File: rust/app/wxwork/wxw_app_token/wxw_app_token_resolver.rs
use color_eyre::eyre::Result;

use super::wxw_app_token_model::WxwGetConfigSignature;

use super::wxw_app_token_service;

/// 通过 appid, agentid, url 生成企业签名
pub async fn wxw_get_config_signature(
  appid: String,
  agentid: String,
  url: String,
) -> Result<WxwGetConfigSignature> {
    
  let wxw_get_config_signature = wxw_app_token_service::wxw_get_config_signature(
    appid,
    agentid,
    url,
  ).await?;
  
  Ok(wxw_get_config_signature)
}

/// 通过 appid, agentid, url 生成应用签名
pub async fn wxw_get_agent_config_signature(
  appid: String,
  agentid: String,
  url: String,
) -> Result<WxwGetConfigSignature> {
      
  let wxw_get_config_signature = wxw_app_token_service::wxw_get_agent_config_signature(
    appid,
    agentid,
    url,
  ).await?;
  
  Ok(wxw_get_config_signature)
}
